.travis.yml

```diff
@@ -9,5 +9,4 @@ rvm:
   - 2.5.1
   - jruby-9.1.17.0
 before_install:
-  - gem install bundler
-  - gem update bundler
+  - gem install bundler -v '~> 1.16'
```

Comment:
I think this breaks the testing. Also we don't need a version constraint here, since this gem doesn't really have dependency on bundler.

Reply:
>  spec.add_development_dependency "bundler", "~> 1.16"

Yes, I think so. Can We remove bundler version constraint from gemspec?